cpp/include/cudf/io/parquet.hpp

```diff
@@ -515,6 +515,15 @@ class chunked_parquet_reader {
  * @file
  */
 
+/**
+ * @brief Struct used to describe column sorting metadata
+ */
+struct sorting_column {
```

Comment:
If this is a public struct and we are taking it from the user, should we give it some default values?

Reply:
Sure, for the bools at least.